Add support for the nRF54H20 DK board (#70172)
zephyrproject-rtos/zephyr: boards/nordic/nrf54h20dk/doc/index.rst

.. _nrf54h20dk_nrf54h20:

nRF54H20 DK
###########

Overview
********

.. note::

   All software for the nRF54H20 SoC is experimental and hardware availability
   is restricted to the participants in the limited sampling program.

The nRF54H20 DK is a single-board development kit for evaluation and development
on the Nordic nRF54H20 System-on-Chip (SoC).

The nRF54H20 is a multicore SoC with:

* an Arm Cortex-M33 core with DSP instructions, FPU, and Armv8-M Security
  Extensions, running at up to 320 MHz, referred to as the **application core**
* an Arm Cortex-M33 core with DSP instructions, FPU, and Armv8-M Security
  Extensions, running at up to 256 MHz, referred to as the **radio core**.

> 

Reviewer: Shouldn't this describe ppr?

Reviewer 2: we should

Author: Yes, info about PPR and its targets will be added.

Author: @nordicjm How about adding it in a follow-up PR together with the picture that is not available yet?

Reviewer: > @nordicjm How about adding it in a follow-up PR together with the picture that is not available yet?

Works for me

* :abbr:`USB (Universal Serial Bus)`
* :abbr:`WDT (Watchdog Timer)`

.. figure:: img/nrf54h20pdk_nrf54h20.webp
     :align: center
     :alt: nRF54H20 PDK

     nRF54H20 PDK (Credit: Nordic Semiconductor)

> 

Reviewer: No picture?

Author: Not yet. I'll add it when I get one.

and configure all the necessary software. Further information can be
found in :ref:`nordic_segger_flashing`.

To build and program the sample to the nRF54H20 DK, complete the following steps:

> First, connect the nRF54H20 DK to you computer using the IMCU USB port on the DK.

Reviewer 3: ```suggestion
1. Connect the nRF54H20 DK to your computer using the IMCU USB port on the DK.
```

Author: Applied.

> Next, build the sample by running the following command:
> 
> .. zephyr-app-commands::
>    :zephyr-app: samples/hello_world

Reviewer 3: ```suggestion
#. Build the sample by running the following command:

   .. zephyr-app-commands::
      :zephyr-app: samples/hello_world
```

Author: Applied.

>    :board: nrf54h20dk/nrf54h20/cpuapp
>    :goals: build flash

Reviewer 3: ```suggestion
      :board: nrf54h20dk/nrf54h20/cpuapp
      :goals: build flash
```

Author: Applied.
